cloud/deployment/inspect/inspect.go

```diff
@@ -0,0 +1,157 @@
+package inspect
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"time"
+
+	"gopkg.in/yaml.v3"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/config"
+)
+
+type deploymentInfo struct {
+	DeploymentID   string    `mapstructure:"deployment-id"`
+	WorkspaceID    string    `mapstructure:"workspace-id"`
+	ClusterID      string    `mapstructure:"cluster-id"`
+	AirflowVersion string    `mapstructure:"airflow-version"`
+	ReleaseName    string    `mapstructure:"release-name"`
+	DeploymentURL  string    `mapstructure:"deployment-url"`
+	WebserverURL   string    `mapstructure:"webserver-url"`
+	UpdatedAt      time.Time `mapstructure:"updated-at"`
+	Status         string    `mapstructure:"status"`
+}
+
+type deploymentConfig struct {
+	Name              string              `mapstructure:"name"`
+	Description       string              `mapstructure:"description"`
+	ClusterID         string              `mapstructure:"cluster-id"` // this is also in deploymentInfo
+	RunTimeVersion    string              `mapstructure:"runtime-version"`
+	SchedulerAU       int                 `mapstructure:"scheduler-au"`
+	SchedulerReplicas int                 `mapstructure:"scheduler-replicas"`
+	AlertEmails       []string            `mapstructure:"alert-emails"`
+	WorkerQueues      []astro.WorkerQueue `mapstructure:"worker-queues"`
+	AstroVariables    Variables           `mapstructure:"astronomer-variables"`
+}
+
+type PrintableDeployment struct {
+	DeploymentInfo   map[string]interface{} `mapstructure:"deployment-info"`
+	DeploymentConfig map[string]interface{} `mapstructure:"deployment-config"`
+}
+
+type Variables struct {
+	EnvFromFile  string
+	VariableList []astro.EnvironmentVariablesObject `mapstructure:"astronomer-variables"`
```

Comment:
nitpick: why have we defined `mapstructure` tag here? it should have been `yaml` & `json`, I believe as of now it would be printed as `variablelist` in the output

Reply:
Good catch. Let me double check what this list of variables should be printed as.